File: app/src/main/java/com/example/vkwall/data/model/GetWallByID/AttachmentById.java
package com.example.vkwall.data.model.GetWallByID;

import com.google.gson.annotations.SerializedName;

public class AttachmentById {
    @SerializedName("type")
    private String type;
    @SerializedName("photo")
    private PhotoById photo;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public PhotoById getPhoto() {
        return photo;
    }

    public void setPhoto(PhotoById photo) {
        this.photo = photo;
    }

}

File: app/src/main/java/com/example/vkwall/data/model/GetWallByID/AttachmentById.kt
package com.example.vkwall.data.model.GetWallByID

import com.example.vkwall.data.model.ProfileWall.Wall.AudioWall
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class AttachmentById {
    @SerializedName("type")
    var type: String? = null
    @SerializedName("audio")
    @Expose
    var audio: AudioWall? = null
    @SerializedName("photo")
    var photo: PhotoById? = null
}
